### PR TITLE
Incorporate mobile auth modal

### PR DESCRIPTION
**Pull request by xtina-starr**
This addresses: https://artsyproduct.atlassian.net/browse/GROW-472

This PR integrates the mobile auth forms and takes in account whether to render the Mobile or Desktop version.

### CODE REVIEW

File: src/desktop/config.coffee [hunk 1]
```diff
@@ -72,7 +72,7 @@ module.exports =
   METAPHYSICS_ENDPOINT: null
   MOBILE_MEDIA_QUERY: "only screen and (max-width: 640px)"
   MOBILE_URL: 'https://m.artsy.net'
-  NEW_AUTH_MODAL: false
+  NEW_AUTH_MODAL: true
```
Comment (xtina-starr) [inline src/desktop/config.coffee]: should this maybe switched back to false?

File: src/desktop/apps/auth2/client/index.tsx [hunk 2]
```diff
@@ -12,10 +13,12 @@ export const init = () => {
   // Rehydrate data from Server
   const bootstrapData = (window as any).__BOOTSTRAP__
   const el = document.getElementById('react-root')
+  const Component = sd.IS_MOBILE ? MobileAuthStatic : AuthStatic
 
   if (el) {
     // Start app
-    ReactDOM.hydrate(<AuthStatic {...bootstrapData} />, el)
+    // ReactDOM.hydrate(<AuthStatic {...bootstrapData} />, el)
```
Comment (kanaabe) [inline src/desktop/apps/auth2/client/index.tsx]: Can remove this

Comment (xtina-starr) [reply src/desktop/apps/auth2/client/index.tsx]: ah yes!

**Comment by kanaabe** [comment]
This looks good to me! Feel free to merge when ready. You had a stray comment in there

**Comment by xtina-starr** [comment]
This is 💚 now. Going to go ahead and merge.

**Pull request merged**